core/Node.js

```diff
@@ -1396,7 +1396,11 @@ Node.prototype.update = function update (time){
 
     while (queue.length) {
         item = this._components[queue.shift()];
-        if (item && item.onUpdate) item.onUpdate(time);
+        if (item && item.onUpdate) {
+            item.onUpdate(time);
+
+            if (item.onUpdated) item.onUpdated(time);
```

Comment:
`onUpdated` is being called on the component after the `onUpdate` method of the component has been called without requiring the component to have an update method.

The components `_requestUpdate` method sets `this._requestingUpdate` to `true`. After the component has been updated, we need to clear this flag again. This would need to happen after the component has been updated. This would mean that the component would have to have an `onUpdate` method. I wanted to avoid the need to call call the super classes onUpdate method when subclassing the component.